p/scripts/user.query.js

```diff
@@ -0,0 +1,54 @@
+// @license magnet:?xt=urn:btih:0b31508aeb0634b347b8270c7bee4d411b5d4109&dn=agpl-3.0.txt AGPL-3.0
+"use strict";
+/* jshint esversion:6, strict:global */
+
+function init_draggable() {
+	if (!window.context) {
+		if (window.console) {
+			console.log('FreshRSS user query waiting for JS…');
+		}
+		setTimeout(init_draggable, 50);
+		return;
+	}
+	
+	let source;
+	const configureQueries = document.querySelector('#configureQueries');
+
+	configureQueries.addEventListener('dragstart', event => {
+		source = event.target.closest('[draggable="true"]');
+		event.dataTransfer.setData('text/html', source.outerHTML);
+		event.dataTransfer.effectAllowed = 'move';
+	});
+	configureQueries.addEventListener('dragover', event => {
+		event.preventDefault();
+	});
```

Comment:
With the arrow function syntax, this can be simplified further to something like:

```suggestion
	configureQueries.addEventListener('dragover', event => event.preventDefault());
```

Reply:
One thing I like about the brackets (and why I don't like optionally leaving them out, at least in the traditional if/else type locations) is that you can just put in another line without any bother. Even if it's just a `console.log(event)`.

Just something to potentially keep in mind. Should be fine here.